sql/updates/world/life_without_regrets.sql

```diff
@@ -0,0 +1,280 @@
+(@CREATUREGUID + 12, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 14, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 18, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 19, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 21, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 22, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 23, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 25, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 27, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 29, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 32, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 35, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 33, 0, 0, 1, 0, 0, ""),
+(@CREATUREGUID + 3, 0, 0, 1, 0, 0, ""),
+
+-- sleeping
+(@CREATUREGUID + 5, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 7, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 9, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 13, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 15, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 16, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 17, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 20, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 24, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 26, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 28, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 30, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 31, 0, 0, 3, 0, 0, "42648"),
+(@CREATUREGUID + 34, 0, 0, 3, 0, 0, "42648");
+
+-- pathing and mounts
+DELETE FROM `creature_addon` WHERE `guid` IN (@CREATUREGUID + 37, @CREATUREGUID + 38, @CREATUREGUID + 39, @CREATUREGUID, @CREATUREGUID + 2, @CREATUREGUID + 1, @CREATUREGUID + 51, @CREATUREGUID + 48, @CREATUREGUID + 49, @CREATUREGUID + 50);
+INSERT INTO `creature_addon` VALUES
+(@CREATUREGUID + 37, (@CREATUREGUID + 37) * 10, 0, 0, 0, 0, ""),
+(@CREATUREGUID + 38, (@CREATUREGUID + 38) * 10, 0, 0, 0, 0, ""),
+(@CREATUREGUID + 39, (@CREATUREGUID + 39) * 10, 0, 0, 0, 0, ""),
+(@CREATUREGUID, @CREATUREGUID * 10, 0, 0, 0, 0, ""),
+(@CREATUREGUID + 2, (@CREATUREGUID + 2) * 10, 0, 0, 0, 0, ""),
+(@CREATUREGUID + 1, (@CREATUREGUID + 1) * 10, 0, 0, 0, 0, ""),
+(@CREATUREGUID + 51, (@CREATUREGUID + 51) * 10, 29283, 0, 0, 0, ""),
+(@CREATUREGUID + 50, (@CREATUREGUID + 50) * 10, 29283, 0, 0, 0, ""),
+(@CREATUREGUID + 48, (@CREATUREGUID + 48) * 10, 29260, 0, 0, 0, ""),
+(@CREATUREGUID + 49, (@CREATUREGUID + 49) * 10, 29260, 0, 0, 0, "");
+
+DELETE FROM `gameobject_addon` WHERE `guid` = @GOBJECTGUID;
+INSERT INTO `gameobject_addon` VALUES
+(@GOBJECTGUID, 7, 1000);
+
+DELETE FROM `spell_area` WHERE `spell` IN (60943, 59062);
+INSERT INTO `spell_area` VALUES
+(60943, 4129, 13257, 13257, 0, 690, 2, 1, 2, 2),
+(59062, 14, 13257, 13267, 0, 690, 2, 1, 2, 11),
+(59062, 1637, 13257, 13267, 0, 690, 2, 1, 2, 11);
+
+DELETE FROM `spell_script_names` WHERE `spell_id` = @OG_TELEPORTSPELL;
+INSERT INTO `spell_script_names` VALUES
+(@OG_TELEPORTSPELL, 'spell_59064_portal_orgrimmar');
+
+-- Its not sniff verified, if anyone got any info about it, you're more than welcome to correct it
+DELETE FROM `spell_target_position` WHERE `ID` = 58419;
+INSERT INTO `spell_target_position` VALUES
```

Comment:
The next time you update your source, you need to insert a LineFeed or NewLine at the end of this file. (I put this comment on a line above in case you don't add more lines to this, so this gets overwritten when you update the file.)